RetailCoder.VBE/ToDoItems/ToDoItem.cs

```diff
@@ -27,6 +28,9 @@ public class ToDoItem
         private readonly int _lineNumber;
         public int LineNumber { get { return _lineNumber; } }
```

Comment:
Isn't the `LineNumber` redundant, given the `QualifiedSelection`? It shortens the binding string I guess, does it?